scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt

```diff
@@ -225,15 +228,15 @@ private fun assertCurrentTime(time: Instant) {
  * Read the template for a ClearlyDefines definitions request from the test file.
  */
 private fun readDefinitionsTemplate(): String {
-    val templateFile = File("src/test/assets/cd_definitions.json")
+    val templateFile = File("$TEST_FILES_ROOT/cd_definitions.json")
     return templateFile.readText()
 }
 
 class ClearlyDefinedStorageTest : WordSpec({
     val wiremock = WireMockServer(
         WireMockConfiguration.options()
             .dynamicPort()
-            .usingFilesUnderDirectory("src/test/assets/")
+            .usingFilesUnderDirectory(TEST_FILES_ROOT)
```

Comment:
Is this configuration option needed here? It does not seem to be defined for the other tests.

Reply:
I believe it's needed (as we use "assets", not "resources"), and I also see `usingFilesUnderDirectory` being called in other tests like `VulnerableCodeTest`, `JiraNotifierTest` etc.